setup/bindir/cloud-get-vm-data-configdrive.in

```diff
@@ -21,23 +21,28 @@ mountdir=$(mktemp -d)
 filepath=$mountdir/cloudstack
 
 user_data=$filepath/userdata/user_data.txt
-availability_zone=$filepath/metadata/availability_zone.txt
```

Comment:
This assumes that bash is available, will the syntactic changes work with `sh`?

Reply:
The original script was only working under bash as well. function is a keyword unknown in `sh`.
As specified in the description, the changes require Bash 4.
I can also add a version specifically for bash 3, but I suppose with shellshock most people will have upgraded to bash 4 by now.